### PR TITLE
zh-trans update content/zh/docs/concepts/workloads/controllers/deploy…

### PR DESCRIPTION
**Pull request by ten2ton**
zh-trans update content/zh/docs/concepts/workloads/controllers/deploy…

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -1,719 +1,1488 @@
...
+描述部署中的 _期望状态_，并且部署控制器以受控速率将实际状态更改为期望状态。可以定义部署以创建新的副本集，或删除现有部署，并通过新的部署使用其所有资源。
 
 {{< note >}}
-You should not manage ReplicaSets owned by a Deployment. All the use cases should be covered by manipulating the Deployment object. Consider opening an issue in the main Kubernetes repository if your use case is not covered below.
+<!--
+
+Do not manage ReplicaSets owned by a Deployment. Consider opening an issue in the main Kubernetes repository if your use case is not covered below.
+
+-->
+不要管理部署拥有的副本集。如果下面未介绍的用例，请考虑在主 Kubernetes 仓库中提出 issue。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/deployment.md]:
```suggestion
不要管理 Deployment 所拥有的 ReplicaSet。如果下面的讨论没有覆盖你的使用场景，请考虑在主 Kubernetes 仓库中提出 issue。
```

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -1,719 +1,1488 @@
...
-
-Here is an example Deployment. It creates a ReplicaSet to bring up three nginx Pods.
+<!--
 
-{{< codenew file="controllers/nginx-deployment.yaml" >}}
+The following are typical use cases for Deployments:
 
-Run the example by downloading the example file and then running this command:
+-->
+以下是典型的部署用例：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/deployment.md]:
```suggestion
以下是 Deployment 的典型用例：
```

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -1,719 +1,1488 @@
...
+<!--
 
-Setting the kubectl flag `--record` to `true` allows you to record current command in the annotations of
-the resources being created or updated. It is useful for future introspection: for example, to see the
-commands executed in each Deployment revision.
+* [Create a Deployment to rollout a ReplicaSet](#creating-a-deployment). The ReplicaSet creates Pods in the background. Check the status of the rollout to see if it succeeds or not.
 
-Then running `get` immediately will give:
+-->
+* [创建部署以展开副本集](#creating-a-deployment)。副本集在后台创建 Pods。检查副本集的状态，查看其是否成功。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/deployment.md]:
```suggestion
* [创建 Deployment 以将 ReplicaSet 上线](#creating-a-deployment)。ReplicaSet 在后台创建 Pods。检查 ReplicaSet 的状态，查看其是否成功。
```

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 4]
```diff
@@ -1,719 +1,1488 @@
...
-NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-nginx-deployment   3         0         0            0           1s
-```
+* [Declare the new state of the Pods](#updating-a-deployment) by updating the PodTemplateSpec of the Deployment. A new ReplicaSet is created and the Deployment manages moving the Pods from the old ReplicaSet to the new one at a controlled rate. Each new ReplicaSet updates the revision of the Deployment.
 
-This indicates that the Deployment's number of desired replicas is 3 (according to deployment's `.spec.replicas`),
-the number of current replicas (`.status.replicas`) is 0, the number of up-to-date replicas (`.status.updatedReplicas`)
-is 0, and the number of available replicas (`.status.availableReplicas`) is also 0.
+-->
+* [声明 Pod 的新状态](#updating-a-deployment) 通过更新部署的 PodTemplateSpec。将创建新的副本集，并且部署管理器以受控速率将 Pod 从旧副本集移动到新副本集。每个新的副本集都会更新部署的修改。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/deployment.md]:
```suggestion
* [声明 Pod 的新状态](#updating-a-deployment)，这是通过更新 Deployment 的 PodTemplateSpec 来完成的。新的 ReplicaSet 会被创建，Deployment 以受控速率将 Pods 从旧的 ReplicSet 移动到新的 ReplicaSet。每个新的 ReplicaSet 都会更新 Deployment 的版本号。
```

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 5]
```diff
@@ -1,719 +1,1488 @@
...
-To see the Deployment rollout status, run:
+* [Rollback to an earlier Deployment revision](#rolling-back-a-deployment) if the current state of the Deployment is not stable. Each rollback updates the revision of the Deployment.
 
-```shell
-$ kubectl rollout status deployment/nginx-deployment
-Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
-deployment "nginx-deployment" successfully rolled out
-```
+-->
+* [回滚到较早的部署版本](#rolling-back-a-deployment)，如果部署的当前状态不稳定。每次回滚都会更新部署的修改。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/deployment.md]:
```suggestion
* [回滚到较早的 Deployment 版本](#rolling-back-a-deployment)：这是在 Deployment 当前状态不稳定时可以执行的操作。每次回滚都会更新 Deployment 的版本。
```

File: content/zh/docs/concepts/workloads/controllers/deployment.md [hunk 6]
```diff
@@ -1,719 +1,1488 @@
...
 {{% capture overview %}}
 
+<!--
+
 A _Deployment_ controller provides declarative updates for [Pods](/docs/concepts/workloads/pods/pod/) and
 [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/).
 
-You describe a _desired state_ in a Deployment object, and the Deployment controller changes the actual state to the desired state at a controlled rate. You can define Deployments to create new ReplicaSets, or to remove existing Deployments and adopt all their resources with new Deployments.
+-->
+一个 _部署_ 控制器为 [Pods](/docs/concepts/workloads/pods/pod/)和 [副本集](/docs/concepts/workloads/controllers/replicaset/)提供描述性的更新方式。
```
[the remark this answers is not in the document]
Comment (ten2ton) [reply content/zh/docs/concepts/workloads/controllers/deployment.md]: Thanks for your review, it has been updated.

**Pull request merged**